public/components/common/helpers/ppl_docs/commands/syntax.ts

```diff
@@ -5,23 +5,17 @@
 
 export const syntaxCmd = `## Syntax
 ---
-### Command Order
+### Command order
 
-The PPL query started with \`search\` command to reference a table search
-from. All the following command could be in any order. In the following
-example, \`search\` command refer the accounts index as the source, then
-using fields and where command to do the further processing.
+The PPL query starts with a \`search\` command to reference a table to search.
+Commans cna be in any order. For example, in the following query, the \`search\` command referencesthe \`accounts\` index as the source, then uses fields and a \`where\` command to do further processing. 
 
 \`\`\` 
 search source=accounts
 | where age > 18
 | fields firstname, lastname
 \`\`\`
 
-### Required arguments
+### Required and optional arguments
 
-Required arguments are shown in angle brackets \< \>.
-
-### Optional arguments
-
-Optional arguments are enclosed in square brackets \[ \].`;
\ No newline at end of file
+Required arguments are enclosed in angle brackets \< \>, and optional arguments are enclosed in square brackets \[ \].`;
```

Comment:
I'll check this usage. This is a TypeScript file, not markdown, so the formatting for code font is different. I think this may be incorrect punctuation.

Reply:
It's closing punctuation. I revised accordingly.